src/main/java/seedu/address/logic/commands/ShowStatisticCommand.java

```diff
@@ -8,7 +8,7 @@
  */
 public class ShowStatisticCommand extends Command {
 
-    public static final String COMMAND_WORD = "showstatistics";
+    public static final String COMMAND_WORD = "stats";
```

Comment:
shall we follow the same convention to be consistent?  verb + action? like show-s?